src/actions/general.js

```diff
@@ -9,11 +10,18 @@ import {loadRolesIfNeeded} from './roles';
 import {logError} from './errors';
 import {batchActions} from 'redux-batched-actions';
 
-export function getPing() {
-    return async (dispatch, getState) => {
-        dispatch({type: GeneralTypes.PING_REQUEST}, getState);
+import type {GeneralState} from '../types/general';
+import type {GetStateFunc, DispatchFunc, ActionFunc} from '../types/actions';
+type response = {
```

Comment:
Probably I would move it to `types`, and change the name, If this is a generic response from the cline, I would name it as "GenericClientResponse" or something like that. We can change it in the future and use specific responses for each client4.js responses.